terraform/digitalocean/main.tf

```diff
@@ -60,26 +60,35 @@ module "etcd_cert" {
   ca_private_key_pem = "${module.ca.ca_private_key_pem}"
 }
 
-module "apiserver_certs" {
-  source             = "../certs/kubernetes/apiserver"
-  ca_cert_pem        = "${module.ca.ca_cert_pem}"
-  ca_private_key_pem = "${module.ca.ca_private_key_pem}"
-  ip_addresses       = "${join(",", digitalocean_droplet.master.*.ipv4_address)}"
-  master_count       = "${var.masters}"
-}
-
-module "worker_certs" {
-  source             = "../certs/kubernetes/worker"
-  ca_cert_pem        = "${module.ca.ca_cert_pem}"
-  ca_private_key_pem = "${module.ca.ca_private_key_pem}"
-  ip_addresses       = "${join(",", digitalocean_droplet.worker.*.ipv4_address)}"
-  worker_count       = "${var.workers}"
-}
-
-module "admin_cert" {
-  source             = "../certs/kubernetes/admin"
-  ca_cert_pem        = "${module.ca.ca_cert_pem}"
-  ca_private_key_pem = "${module.ca.ca_private_key_pem}"
+module "kube_apiserver_certs" {
+  source                = "github.com/Capgemini/tf_tls/kubernetes/apiserver"
+  ca_cert_pem           = "${module.ca.ca_cert_pem}"
+  ca_private_key_pem    = "${module.ca.ca_private_key_pem}"
+  ip_addresses          = "${join(",", digitalocean_droplet.master.*.ipv4_address)}"
```

Comment:
In the module we are doing     ```${element(split(\",\", var.ip_addresses), count.index)}"```
So would probably make sense just pass ``` digitalocean_droplet.master.*.ipv4_address```as the parameter and remove the transformation inside the module